catberry_components/todo/list/template.hbs

```diff
@@ -1,16 +1,16 @@
 <ul id="todo-list">
 	<!-- These are here just to show the structure of the list items -->
 	<!-- List items should get the class `editing` when editing and `completed` when marked as completed -->
-	{{#items}}
-		<li {{#if isCompleted}}class="completed"{{/if}} data-index="{{@index}}">
+	{{#each items}}
+		<li {{#if this.isCompleted}}class="completed"{{/if}} data-index="{{@key}}">
```

Comment:
`@key` is a handlebars helper containing key from current `K => V` pair. So I can't get why it should be renamed.